api_spec.yaml

```diff
@@ -1231,11 +1231,217 @@ paths:
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            type: object
+                            properties:
+                                tenantId:
+                                    type: string
+                                emailPasswordEnabled:
+                                    type: boolean
+                                passwordlessEnabled:
+                                    type: boolean
+                                thirdPartyEnabled:
+                                    type: boolean
+                                coreConfig:
+                                    type: object
+                                firstFactors:
+                                    type: array
+                                    items:
+                                        type: string
+                                    example:
+                                        - "emailpassword"
+                                        - "thirdparty"
+                                requiredSecondaryFactors:
+                                    type: array
+                                    items:
+                                        type: string
+                                    example:
+                                        - "otp-email"
+                            required:
+                                - tenantId
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    status:
+                                        type: string
+                                        default: "OK"
+                                    createdNew:
+                                        type: boolean
+
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
     /dashboard/api/tenants/list:
         get:
             tags:
                 - Multitenancy
             summary: Get all tenants created in the core
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
```

Comment:
was this missing earlier? is this a fix now?

Reply:
Yes